src/extension/app/components/action-bar/menu-item/menu-item.css.js

```diff
@@ -24,6 +24,13 @@ export const style = css`
     font-weight: 700;
   }
 
+  :host(.current-env.env-edit[aria-disabled="true"]) #label,
+  :host(.current-env.env-edit[disabled]) #label,
+  :host(.current-env.env-edit[aria-disabled="true"]) [name="description"]::slotted(*),
+  :host(.current-env.env-edit[disabled]) [name="description"]::slotted(*) {
+    color: var(--mod-menu-item-label-content-color-default);
```

Comment:
@rofe Curious where this variable comes from?

Reply:
@dylandepass Read them out of spectrum CSS 😬 happy to change to something more appropriate of course

Reply:
removed the `mod-` prefix, looks the same without